src/DataAPI.jl

```diff
@@ -290,97 +290,128 @@ function allcombinations end
 const STYLE_INFO = """
 One of the uses of the metadata style information is decision
 how the metadata should be propagated when `x` is transformed. This interface
-defines `:none` style that indicates that metadata should not be propagated
+defines the `:none` style that indicates that metadata should not be propagated
 under transformations. At least this style must be supported by any type
 defining support for metadata.
 """
 
+const COL_INFO = """
+`col` must be a type that is supported by table `x`. Following the Tables.jl
```

Comment:
Here and in other docstrings, I would rather give information intended at users than intended at package developers, as docstrings are visible to all users. This is mainly a matter of formulation, i.e. say what one can do with the API rather how one is allowed to implement it.

Reply:
I have updated the docstrings